.gitignore

```diff
@@ -0,0 +1 @@
+.github/workflows/linters.yml
```

Comment:
- [ ] [Optional] This file should not be ignored. Kindly remove it from the .gitignore file

Reply:
Yes sure.

I put it because my mentor told me that you can put this file to git ignore to prevent any other issues.